ingestion_tools/dataset_configs/10301.yaml

```diff
@@ -270,7 +283,7 @@ runs:
       match_regex: /.*_BrnoKrios_.*$
       name_regex: (.*)
 standardization_config:
-  deposition_id: 10301
+  deposition_id: 10300
```

Comment:
We can probably do away with this field now?

Reply:
I can remove this, after I update the ingestion PR to source the deposition_id from the parent instead of standardization_config.